stablehlo/tools/StablehloTranslateMain.cpp

```diff
@@ -131,15 +156,29 @@ llvm::Error interpreterFallback(Operation &op, stablehlo::Scope &scope,
 TranslateFromMLIRRegistration interpretRegistration(
     "interpret", "Interpreter for StableHLO",
     [](ModuleOp module, raw_ostream &os) {
+      auto numFuncs = 0;
+      bool isDistributed = false;
```

Comment:
hasMain? I can imagine wanting to interpret a non-distributed program with multiple functions, e.g. one that has a custom call with called computations.